src/Consolonia.Core/Drawing/DrawingContextImpl.cs

```diff
@@ -547,8 +554,21 @@ private void FillRectangleWithBrush(IBrush brush, Rect r)
                             _pixelBuffer[x, y] = _pixelBuffer[x, y].Invert();
                             break;
                         default:
+                            Pixel pixelAbove;
+                            if (solidColorBrush == null)
```

Comment:
good change, this was on my list to change. I had noticed we were calling that over and over.